src/senaite/sync/utils.py

```diff
@@ -28,6 +30,22 @@ def to_review_history_format(review_history):
     return review_history
 
 
+def is_item_allowed(item):
+    """
+    Check if an item can be handled based in its portal type.
+    :return:
+    """
```

Comment:
Please take care of your docstrings. Use a consistent format for the whole package and fill out `:return:` if you use it.